packages/atlas/src/components/_overlays/ChangePriceDialog/ChangePriceDialog.tsx

```diff
@@ -1,22 +1,36 @@
 import styled from '@emotion/styled'
 import BN from 'bn.js'
-import { FC, useState } from 'react'
+import { FC, useEffect, useState } from 'react'
 
+import { Fee } from '@/components/Fee'
 import { Text } from '@/components/Text'
 import { TokenInput } from '@/components/_inputs/TokenInput'
 import { DialogModal } from '@/components/_overlays/DialogModal'
 import { tokenNumberToHapiBn } from '@/joystream-lib/utils'
+import { useFee } from '@/providers/joystream'
 import { sizes } from '@/styles'
 
 type ChangePriceDialogProps = {
   onModalClose: () => void
   isOpen: boolean
   onChangePrice: (id: string, price: BN) => void
   nftId: string | null
+  memberId: string | null
 }
 
-export const ChangePriceDialog: FC<ChangePriceDialogProps> = ({ onModalClose, isOpen, onChangePrice, nftId }) => {
-  const [price, setPrice] = useState<number | null>()
+export const ChangePriceDialog: FC<ChangePriceDialogProps> = ({
+  onModalClose,
+  isOpen,
+  onChangePrice,
+  nftId,
+  memberId,
+}) => {
+  const [price, setPrice] = useState<number | null>(null)
+  const amountBn = tokenNumberToHapiBn(price || 0)
+  const { fullFee, loading: feeLoading } = useFee(
+    'changeNftPriceTx',
+    memberId && nftId ? [memberId, nftId, amountBn.toString()] : undefined
+  )
```

Comment:
Should we also add `isOpen` to the list of conditions for calculating this? Then fee isn't loaded until you actually want to change the price